README.md

```diff
@@ -49,7 +49,6 @@ The most difficult part of automated testing on mobile is the tip of the testing
 - **Runs on Devices** (not yet supported on iOS): Gain confidence to ship by testing your app on a device/simulator just like a real user.
 - **Automatically Synchronized:** Stops flakiness at the core by monitoring asynchronous operations in your app.
 - **Made For CI:** Execute your E2E tests on CI platforms like Travis without grief.
-- **Test Runner Independent:** Use Jest, Mocha, AVA, or any other JavaScript test runner you like (spoiler: we have our favorite).
```

Comment:
@d4vidi  I had to do it. 🤷 
@asafkorem , @jonathanmos 

If you can come up with another selling statement... like "using the most natural test runner for React Native which is Jest, mu-ha-ha" 😈

Reply:
This is quite alright, as it aligns with what we've agreed upon. We will be reiterating this, of course 😅 
Nevertheless, there's still one more place to clean up: https://github.com/wix/Detox/blob/next/website/src/components/HomepageFeatures.js#L11

![image](https://user-images.githubusercontent.com/9818880/167303076-0544dcb9-268a-439f-bc50-3dc4173d5b20.png)

@asafkorem this is the place jump in on the discussion 😄

Reply:
This is quite alright, as it aligns with what we've agreed upon. We will be reiterating this, of course 😅 
Nevertheless, there's still one more place to clean up: https://github.com/wix/Detox/blob/next/website/src/components/HomepageFeatures.js#L11

![image](https://user-images.githubusercontent.com/9818880/167303076-0544dcb9-268a-439f-bc50-3dc4173d5b20.png)

@asafkorem this is the place jump in on the discussion 😄

Reply:
@noomorph I suggest to replace it with _"Supports Multiple Workers" / "Supports Parallel Testing"_ (description can be smth like: _"Run parallel tests with multiple workers."_)
We can keep the current icon (it fits), or use https://uxwing.com/engineer-workers-icon/. WDYT?

Reply:
I mean, to replace the outdated homepage section. To the readme file we can add both bullets (parallel testing & using Jest)

Reply:
Brilliant idea! Thanks for your $0.02, Asaf - indeed, it fits!